index.js

```diff
@@ -146,31 +182,41 @@ function nowServing(line) {
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
+let i = 0
+function takeATicketNumber(line){
+i ++
+line.push(i) 
+return `Welcome. You are ticket number ${i}`
+ }
```

Comment:
Nice! Just make sure to indent your code so it's easy to see what's inside the function and what's not.